xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -521,13 +521,20 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
     const CSettingControlTitle *title = dynamic_cast<CSettingControlTitle *>((*groupIt)->GetControl());
     bool hideSeparator = title ? title->IsSeparatorHidden() : false;
```

Comment:
well, it doesn't make much sense to show a separator if there is nothing to separate it from :)